expressEngine/lib/service/db.js

```diff
@@ -14,6 +14,11 @@ class DbOperation {
   findOne = (collection, data = {}, projection = {}) => {
     return collection.findOne(data, projection);
   }
+  findAll = ({collection, data = {}, skip = 0, limit = 10 , projection = {}}) => {
+    const skipValue = isNaN(skip) ? 0 : parseInt(skip, 10);
```

Comment:
Create a generic method for converting string to number, pass two arguments (value, defaultValue);

Reply:
done